CMakeLists.txt

```diff
@@ -117,6 +117,22 @@ if(MSVC OR MSVC_IDE)
     endif()
 else()
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic -Wextra -Wno-unknown-pragmas -std=c++11")
+    if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
```

Comment:
`CMAKE_COMPILER_IS_GNUCXX` ?

Reply:
CMake now recommends using `CMAKE_CXX_COMPILER_ID` instead https://cmake.org/cmake/help/latest/variable/CMAKE_COMPILER_IS_GNUCXX.html

Reply:
oh good to know, thanks :+1: